pages/products/products.css

```diff
@@ -53,7 +53,9 @@ body {
 }
 
 .sort-by-title {
-  margin: 1rem 0 1rem 0;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  margin: 1rem 0 1rem;
```

Comment:
why multiple margin values margin top bottom than margin alone ?

Reply:
forgot to remove margin top and bottom